doc/letter_count/input.txt

```diff
@@ -1,21 +1,2 @@
-Quod equidem non reprehendo;
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quibus natura iure responderit non esse verum aliunde finem beate vivendi, a se principia rei gerendae peti; Quae enim adhuc protulisti, popularia sunt, ego autem a te elegantiora desidero. Duo Reges: constructio interrete. Tum Lucius: Mihi vero ista valde probata sunt, quod item fratri puto. Bestiarum vero nullum iudicium puto. Nihil enim iam habes, quod ad corpus referas; Deinde prima illa, quae in congressu solemus: Quid tu, inquit, huc? Et homini, qui ceteris animantibus plurimum praestat, praecipue a natura nihil datum esse dicemus?
```

Comment:
I don't understand why github thinks this file used to be Lorem ipsum content. It was newly created for this PR. Bewildering.

Reply:
Oh I see what happened. nvm